docs/executor-artillery.md

```diff
@@ -0,0 +1,170 @@
+# Artillery.io performance tests
+
+
+## Test Environment
+
+Let's assume that our SUT (Service Under Test) is an internal Kubernetes service which has
+ClusterIP `Service` created and is exposed on port `8088`. The service name is `testkube-api-server`
+and is exposing the `/health` endpoint that we want to test.
+
+To call the SUT inside a cluster:
+
+```sh
+curl http://testkube-api-server:8088/health
+```
+
+Output:
+
+```sh
+200 OK
+```
+
+## Create a test manifest
+
+The Artillery tests are defined in declarative manner, as YAML files.  
+Test should warm up our service a little bit first, next we can hit a little harder. 
+
+Let's save our test into `test.yaml` file with content below:   
+
+```yaml
+config:
+  target: "http://testkube-api-server:8088"
+  phases:
+    - duration: 6
+      arrivalRate: 5
+      name: Warm up
+    - duration: 120
+      arrivalRate: 5
+      rampTo: 50
+      name: Ramp up load
+    - duration: 60
+      arrivalRate: 50
+      name: Sustained load
+scenarios:
+  - name: "Check health endpoint"
+    flow:
+      - get:
+          url: "/health"
+```
+
+Our test is ready, but how to run it in Kubernetes cluster? Testkube will help you with that! 
```

Comment:
Our test is ready, but how do we run it in a Kubernetes cluster? Testkube will help you with that!